packages/commons/src/countryconfig/openapi.ts

```diff
@@ -29,26 +28,8 @@ const openapi = new OpenApiBuilder({
 
   .build()
 
+const clean = JSON.parse(
+  JSON.stringify(openapi, (_, v) => (v === null ? undefined : v))
+)
```

Comment:
Why is this done, does the output contain some null values that are cleaned away?